Source/Dafny/Compilers/Compiler-python.cs

```diff
@@ -87,10 +120,29 @@ public class PythonCompiler : SinglePassCompiler {
 
       if (dt is TupleTypeDecl) {
         return null;
-      } else {
-        throw new NotImplementedException();
       }
 
+      var DtT = dt.CompileName;
+
+      var btw = wr.NewBlockPy($"class {DtT}:");
+
+      foreach (var ctor in dt.Ctors) {
+        var namedtuple = $"NamedTuple(\"{ctor.CompileName}\", [])";
```

Comment:
I'd suggest using `namedtuple` instead of `NamedTuple` here, for compatibility with Python < 3.10.